dso-l2/src/main/java/com/tc/objectserver/handshakemanager/ServerClientHandshakeManager.java

```diff
@@ -286,7 +288,7 @@ synchronized int getUnconnectedClientsSize() {
     private ReconnectTimerTask(ServerClientHandshakeManager handshakeManager, Timer timer) {
       this.handshakeManager = handshakeManager;
       this.timer = timer;
-      this.timeToWait = handshakeManager.reconnectTimeout;
+      this.timeToWait = handshakeManager.reconnectTimeoutSupplier.get();
```

Comment:
`ReconnectTimerTask` and `ServerClientHandshakeManager` might not have the same timeout value, I think its better to create the ` ReconnectTimerTask` task  in `startReconnectWindow` passing the timeout value instead of creating in the constructor

Reply:
Done